website/shortlink/models.py

```diff
@@ -0,0 +1,33 @@
+from django.conf import settings
+from django.core import validators
+from django.db import models
+
+from django.utils.translation import gettext_lazy as _
+
+
+class ShortLink(models.Model):
+    key = models.CharField(
+        primary_key=True,
+        max_length=32,
+        validators=[validators.validate_slug],
+        verbose_name=_("Name for the url"),
+        help_text=_(f"This is what you use after https://{settings.SITE_DOMAIN}/"),
+    )
+
+    url = models.TextField(
```

Comment:
This can just be a URLField https://docs.djangoproject.com/en/3.1/ref/models/fields/#urlfield